### ISSUE TITLE
Process killed when trying to install pypi dependency

### ISSUE THREAD

**Issue by tadejsv** [prefix-dev/pixi#938]
### Checks

- [X] I have checked that this issue has not already been reported.

- [X] I have confirmed this bug exists on the [latest version](https://github.com/prefix-dev/pixi/releases) of pixi, using `pixi --version`.

### Reproducible example

Create this `pixi.toml` file

```toml
[project]
name = "k"
version = "0.1.0"
description = "Add a short description here"
authors = ["Tadej <tadej.svetina@gmail.com>"]
channels = ["conda-forge"]
platforms = ["linux-64"]

[tasks]

[dependencies]
python = "3.11.*"
pip = "24.0.*"

[pypi-dependencies]
boto3-stubs = {version = "==1.*", extras=["essential"]}
```

Then, try to intall with
```
pixi install
```

The output is
```
 WARN pixi::project::manifest: BETA feature `[pypi-dependencies]` enabled!

Please report any and all issues here:

        https://github.com/prefix-dev/pixi.

Turn this warning off by setting the environment variable `PIXI_BETA_WARNING_OFF` to `true`.

⠦ updating lock-file   [00:00:09] [━━━━━━━━━━━━━━━━━━━━━━━━╾───────────────]    3/5                                                                                                               
  ⠓      default: linux-64 [00:00:08] resolving pypi dependencies   
[1]    1131902 killed     pixi install
```

### Issue description

Pixi install gets killed trying to intall a pypi dependency

### Expected behavior

Should not get killed and install correctly

**Comment by ruben-arts** [comment]
He @tadejsv, I can't recreate it on main. We now use `uv` on main. And it will be released in the next few minutes. Hopefully that fixes it for you!

**Comment by tadejsv** [comment]
@ruben-arts Can confirm it is fixed in 0.16.0!
